tests/test_server_basic.py

```diff
@@ -93,6 +93,41 @@ def test_empty_request(dispatcher_live_fixture):
     logger.info(jdata['config'])
 
 
+def test_no_debug_mode_empty_request(dispatcher_live_fixture_no_debug_mode):
+    server = dispatcher_live_fixture_no_debug_mode
+    print("constructed server:", server)
+
+    c=requests.get(server + "/run_analysis",
+                   params={},
+                )
+
+    print("content:", c.text)
+
+    jdata=c.json()
+
+    assert c.status_code == 400
+
+     # parameterize this
+    assert jdata['installed_instruments'] == [] or \
+           jdata['installed_instruments'] == ['spi_acs']
```

Comment:
is this one still needed? we might have moved all plugin tests out?

Reply:
ops I will adjust it